cmake/adjust_global_compile_flags.cmake

```diff
@@ -131,6 +131,7 @@ if (onnxruntime_DISABLE_RTTI)
     # Disable RTTI and turn usage of dynamic_cast and typeid into errors
     add_compile_options("$<$<COMPILE_LANGUAGE:CXX>:/GR->" "$<$<COMPILE_LANGUAGE:CXX>:/we4541>")
   else()
+    # Avoid unboundTypeError occurring in embind for pointers to primitives
```

Comment:
Use: "Avoid unboundTypeError since unbound type names are illegal with RTTI disabled in Embind API"